qiskit_experiments/data_processing/data_processor.py

```diff
@@ -168,3 +171,29 @@ def train(self, data: List[Dict[str, Any]]):
                         train_data.append(self._call_internal(datum, call_up_to_node=index)[0])
 
                     node.train(train_data)
+
+    @classmethod
+    def get_processor(
```

Comment:
This implicitly assumes we need data processor that returns non-classified outcome. What happens if we need count, population, or something except for iq data format? We add class method for every types of outcome? Perhaps we need smarter logic that automatically assemble processor depending on the format of outcome. Or, we can prepare data processor library (i.e. correction of functions that generates data processor). I think latter approach is better since we can avoid managing complicated logic here.

Reply:
Good point. I started a processor library here 9b14031 which could be a collection of methods to return data processors.